content/id/docs/concepts/configuration/secret.md

```diff
@@ -0,0 +1,1062 @@
+dan kemudian mengaplikasikannya untuk membuat sebuah objek pada Apiserver. _Generator_ yang digunakan haruslah 
+dispesifikasikan di dalam sebuah _file_ `kustomization.yaml` di dalam sebuah direktori.
+
+Sebagai contoh, untuk menghasilan sebuah Secret dari _file-file_ `./username.txt` dan `./password.txt`
+```shell
+# Create a kustomization.yaml file with SecretGenerator
+cat <<EOF >./kustomization.yaml
+secretGenerator:
+- name: db-user-pass
+  files:
+  - username.txt
+  - password.txt
+EOF
+```
+Gunakan direktori _kustomization_ untuk membuat objek Secret yang diinginkan.
+```shell
+$ kubectl apply -k .
+secret/db-user-pass-96mffmfh4k created
+```
+
+Kamu dapat memastikan Secret tersebut sudah dibuat dengan menggunakan perintah berikut:
+
+```shell
+$ kubectl get secrets
+NAME                             TYPE                                  DATA      AGE
+db-user-pass-96mffmfh4k          Opaque                                2         51s
+
+$ kubectl describe secrets/db-user-pass-96mffmfh4k
+Name:            db-user-pass
+Namespace:       default
+Labels:          <none>
+Annotations:     <none>
+
+Type:            Opaque
+
+Data
+====
+password.txt:    12 bytes
+username.txt:    5 bytes
+```
+
+Sebagai contoh, untuk membuat sebuah Secret dari literal `username=admin` dan `password=secret`,
+kamu dapat menspesifikasikan _generator_ Secret pada _file_ `kustomization.yaml` sebagai
+```shell
+# Create a kustomization.yaml file with SecretGenerator
+$ cat <<EOF >./kustomization.yaml
+secretGenerator:
+- name: db-user-pass
+  literals:
+  - username=admin
+  - password=secret
+EOF
+```
+Aplikasikan direktori _kustomization_ untuk membuat objek Secret.
+```shell
+$ kubectl apply -k .
+secret/db-user-pass-dddghtt9b5 created
+```
+{{< note >}}
+Secret yang dihasilkan nantinya akan memiliki tambahan sufix dengan cara melakukan teknik _hashing_ 
+pada isi Secret tersebut. Hal ini dilakukan untuk menjamin dibuatnya sebuah Secret baru setiap kali terjadi 
+perubahan isi dari Secret tersebut.
+{{< /note >}}
+
+#### Melakukan Proses _Decode_ pada Secret
+
+Secret dapat dibaca dengan menggunakan perintah `kubectl get secret`. 
+Misalnya saja, untuk membaca Secret yang dibuat pada bagian sebelumya:
+
+```shell
+kubectl get secret mysecret -o yaml
+```
+```
+apiVersion: v1
+kind: Secret
+metadata:
+  creationTimestamp: 2016-01-22T18:41:56Z
+  name: mysecret
+  namespace: default
+  resourceVersion: "164619"
+  selfLink: /api/v1/namespaces/default/secrets/mysecret
+  uid: cfee02d6-c137-11e5-8d73-42010af00002
+type: Opaque
+data:
+  username: YWRtaW4=
+  password: MWYyZDFlMmU2N2Rm
+```
+
+Kemudian lakukan mekanisme _decode_ _field_ _password_:
+
+```shell
+echo 'MWYyZDFlMmU2N2Rm' | base64 --decode
+```
+```
+1f2d1e2e67df
+```
+
+## Menggunakan Secret
+
+Secret dapat di-_mount_ sebagai _volume_ data atau dapat diekspos sebagai 
+{{< glossary_tooltip text="variabel-variabel environment" term_id="container-env-variabeles" >}}
```

Comment:
typo:
```suggestion
{{< glossary_tooltip text="variabel-variabel environment" term_id="container-env-variables" >}}
```
then

add https://github.com/kubernetes/website/blob/master/content/en/docs/reference/glossary/container-env-variables.md to `id/docs/references/glossary/container-env-variables.md`

Reply:
I think you also need to add more files to the `id/docs/references/glossary`